src/main/kotlin/no/nav/hag/utils/bakgrunnsjobb/BakgrunnsjobbRepository.kt

```diff
@@ -76,7 +72,9 @@ class MockBakgrunnsjobbRepository : BakgrunnsjobbRepository {
 
     override fun deleteOldOkJobs(months: Long) {
         val someMonthsAgo = LocalDateTime.now().minusMonths(months)
-        jobs.removeIf { it.behandlet?.isBefore(someMonthsAgo)!! && it.status.equals(BakgrunnsjobbStatus.OK) }
+        jobs.values.filter {  it.behandlet?.isBefore(someMonthsAgo)!! && it.status.equals(BakgrunnsjobbStatus.OK) }.map { it.uuid }.forEach {
+            jobs.remove(it)
+        }
```

Comment:
```suggestion
        jobs.values.filter {  
        it.behandlet!!.isBefore(someMonthsAgo) && it.status == BakgrunnsjobbStatus.OK
        }
        .map { it.uuid }
        .forEach {
            jobs.remove(it)
        }
```
Dette må nok lintes.